apps/etterlatte-beregning/src/main/kotlin/beregning/regler/barnepensjon/BeregnBarnepensjon.kt

```diff
@@ -54,42 +55,34 @@ val beregnBarnepensjon1967Regel =
         sats.multiply(trygdetidsfaktor)
     }
 
-val barnepensjonSatsMedInstitusjonsopphold =
-    RegelMeta(
-        gjelderFra = BP_1967_DATO,
-        beskrivelse = "Sikrer at ytelsen ikke blir større med institusjonsoppholdberegning",
-        regelReferanse = RegelReferanse(id = "BP-BEREGNING-1967-GUNSTIGHET-INSTITUSJON", versjon = "2"),
-    ) benytter barnepensjonSatsRegel og institusjonsoppholdSatsRegel med { standardSats, institusjonsoppholdSats ->
-        institusjonsoppholdSats.coerceAtMost(standardSats)
-    }
-
-val barnepensjonSats =
-    RegelMeta(
-        gjelderFra = BP_1967_DATO,
-        beskrivelse = "Bruker institusjonsoppholdberegning hvis barnet er i institusjon",
-        regelReferanse = RegelReferanse("BP-BEREGNING-1967-KANSKJEANVENDINSTITUSJON", versjon = "2"),
-    ) benytter barnepensjonSatsRegel og barnepensjonSatsMedInstitusjonsopphold og erBrukerIInstitusjon med {
-            satsIkkeInstitusjonsopphold,
-            satsInstitusjonsopphold,
-            harInstitusjonshopphold,
-        ->
-        if (harInstitusjonshopphold) satsInstitusjonsopphold else satsIkkeInstitusjonsopphold
-    }
-
-val beregnBarnepensjon1967RegelMedInstitusjon =
+val beregnBarnepensjon =
     RegelMeta(
         gjelderFra = BP_1967_DATO,
         beskrivelse = "Reduserer ytelsen mot opptjening i folketrygden inkludert institusjonsopphold",
         regelReferanse = RegelReferanse(id = "BP-BEREGNING-1967-REDUSERMOTTRYGDETID-INSTITUSJON", versjon = "2"),
-    ) benytter barnepensjonSats og trygdetidsFaktor med { sats, trygdetidsfaktor ->
+    ) benytter barnepensjonSatsRegel og trygdetidsFaktor med { sats, trygdetidsfaktor ->
         sats.multiply(trygdetidsfaktor)
     }
 
+val beregnGunstigstBarnepensjon =
```

Comment:
Mulig `beregnGunstigstBarnepensjon` bli litt misvisende. Vi gir jo ikke den gunstigste for bruker her når jeg tenker meg om? Hva med `beregnBarnepensjonMedGunstighetsvurderingInstitusjon`, eller noe sånt?

Reply:
Rettferdig, eller skal vi bare droppe å beskrive det?

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5303/commits/3509e42df7f02e558d92eb8176d6231110f83e42